From 761138c4e2cd51989d517026fb147dc700cd0394 Mon Sep 17 00:00:00 2001
From: Vincent Lejeune <vljn.ovi@gmail.com>
Date: Sat, 13 Jan 2018 17:57:39 +0100
Subject: [PATCH 4/7] Export fix

---
 src/caffe/CMakeLists.txt | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/src/caffe/CMakeLists.txt b/src/caffe/CMakeLists.txt
index b9152e92..599a3a5d 100644
--- a/src/caffe/CMakeLists.txt
+++ b/src/caffe/CMakeLists.txt
@@ -36,6 +36,15 @@ set_target_properties(caffe PROPERTIES
     SOVERSION ${CAFFE_TARGET_SOVERSION}
     )
 
+if(MSVC AND BUILD_SHARED_LIBS)
+  # CMake 3.4 introduced a WINDOWS_EXPORT_ALL_SYMBOLS target property that makes it possible to 
+  # build shared libraries without using the usual declspec() decoration. 
+  # See: https://blog.kitware.com/create-dlls-on-windows-without-declspec-using-new-cmake-export-all-feature/ 
+  # and https://cmake.org/cmake/help/v3.5/prop_tgt/WINDOWS_EXPORT_ALL_SYMBOLS.html 
+  # for details. 
+  set_target_properties(caffe PROPERTIES WINDOWS_EXPORT_ALL_SYMBOLS TRUE) 
+endif()
+
 # ---[ Tests
  add_subdirectory(test)
 
-- 
2.14.1.windows.1

