From 1117906b565db5d54767554e16ab0f3f9d372507 Mon Sep 17 00:00:00 2001
From: Vincent Lejeune <vljn.ovi@gmail.com>
Date: Sun, 14 Jan 2018 00:09:46 +0100
Subject: [PATCH 7/8] Fix dependencies.

---
 tools/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 767c43f9..c343cc32 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -12,7 +12,7 @@ foreach(source ${srcs})
 
   # target
   add_executable(${name} ${source})
-  target_link_libraries(${name} ${Caffe_LINK})
+  target_link_libraries(${name} caffe)
   caffe_default_properties(${name})
 
   # set back RUNTIME_OUTPUT_DIRECTORY
-- 
2.14.1.windows.1

